ultet/build.sbt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2022 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Dependencies._
+import com.github.sbt.jacoco.report.JacocoReportSettings
+
+ThisBuild / organization := "za.co.absa.fa-db"
+
+
+lazy val scala212 = "2.12.17"
```

Comment:
I would go for 2.13 since it is standalone tool, if you guys don't mind

Reply:
Not sure, if all eventual libs are available in 2.13
If you believe it won't be an issue, please, change in your PR on the main function.